libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/RoadObjectType.kt

```diff
@@ -0,0 +1,117 @@
+package com.mapbox.navigation.base.trip.model.roadobject
+
+import androidx.annotation.IntDef
+
+/**
+ * Holds available [RoadObject] types.
+ *
+ * Available values are:
+ * - [RoadObjectType.TunnelEntrance]
+ * - [RoadObjectType.TunnelExit]
+ * - [RoadObjectType.Tunnel]
+ * - [RoadObjectType.CountryBorderCrossing]
+ * - [RoadObjectType.TollCollection]
+ * - [RoadObjectType.RestStop]
+ * - [RoadObjectType.RestrictedAreaEntrance]
+ * - [RoadObjectType.RestrictedAreaExit]
+ * - [RoadObjectType.RestrictedArea]
+ * - [RoadObjectType.BridgeEntrance]
+ * - [RoadObjectType.BridgeExit]
+ * - [RoadObjectType.Bridge]
+ * - [RoadObjectType.Incident]
+ * - [RoadObjectType.Custom]
+ */
+object RoadObjectType {
+
+    /**
+     * Type of the [TunnelEntrance].
+     */
+    const val TunnelEntrance = 0
```

Comment:
Should this be Java constant notation instead `TUNNEL_ENTRANCE`? 🤔 

Same thing applies to the rest of the constants 👇

Reply:
I don't see an issue here, we can change it or leave as is.

Reply:
I'd rather to stick to one or the other for consistency. So probably easier and faster to update these ones.

Reply:
done